### ISSUE TITLE
Function _generate_chart_from_df in lwchart.py overwrites resulting .pdf and .png files

### ISSUE THREAD

**Issue by Guido589** [open-s4c/benchkit#19]
Calling `generate_graph` more than once on the same campaign, as in the example below, sometimes causes the resulting `.png` and `.pdf` to be overwritten if the seconds in the timestamp part of the name are the same:

```python
suite.generate_graph(
    plot_name="barplot",
    x="bench_version",
    y="perf-stat/cpu-cycles",
    hue="bench_version",
)

suite.generate_graph(
    plot_name="barplot",
    x="bench_version",
    y="perf-stat/cache-misses",
    hue="bench_version",
)
```
![image0](https://github.com/open-s4c/benchkit/assets/73546496/69761148-deef-46b4-9a76-42d6101b4774)

**Comment by apaolillo** [comment]
Good catch! This is a very annoying one. I'm going to solve that next.

**Comment by apaolillo** [comment]
Fixed by #76
